docker-compose.yml

```diff
@@ -23,6 +23,19 @@ services:
       bahmni:
         aliases:
           - bahmniapps-1
+  implementer-interface:
```

Comment:
@bhiravabhatla Could you insert a empty line here, for clarity?

Reply:
Done